exercises/bracket-push/example.tt

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env ruby
+gem 'minitest', '>= 5.0.0'
+require 'minitest/autorun'
+require_relative 'bracket_push'
+
+# Test data version:
+# <%= sha1 %>
+class BracketsTest < Minitest::Test<% test_cases.each do |test_case| %>
+  def <%= test_case.name %><% if test_case.skipped? %>
```

Comment:
Should we avoid having in the 'view'?  See `xruby/clock/example.tt` along with `lib/clock_cases.rb` with the `skipped` method.